docs/faq.rst

```diff
@@ -92,3 +92,28 @@ preview" status. However, you can take a look at `this blog post`_ outlining a
 workable "DIY" solution as a stop gap for now.
 
 .. _`this blog post`: https://www.jeffgeerling.com/blog/2019/how-add-integration-tests-ansible-collection-molecule
+
+Does Molecule support monorepos?
+================================
+
+Yes, roles contained in a `monorepo`_ with other roles are automatically picked
+up and ``ANSIBLE_ROLES_PATH`` is set accordingly. See `this page`_ for more
+information.
+
+.. _monorepo: https://en.wikipedia.org/wiki/Monorepo
```

Comment:
```suggestion
.. _`monorepo`: https://en.wikipedia.org/wiki/Monorepo
```

Argh, another! ;)

Reply:
Fixed and squashed, rebased on top of `master`.